hil/ext/switches/common.py

```diff
@@ -53,3 +56,36 @@ def parse_vlans(raw_vlans):
             vlan_list.append(num_str)
 
     return vlan_list
+
+
+def modify_port_common(self, port, channel, new_network):
+    """This implements modify port method that non-console like drivers can use
```

Comment:
Rather than "non-console", I might suggest mentioning that it is vlan-specific (we still don't have any other channel types, but this is the constraint).